src/components/View/commons/Topics.jsx

```diff
@@ -26,9 +26,9 @@ const Topics = ({ content }) => {
 
   return list?.length > 0 ? (
     <div className="mt-4 mb-5 page-topics">
-      <h5>
+      <h3 clasName="h5">
```

Comment:
https://iosanita-staging.c3p0.redturtle.it/test-rt-new#:~:text=sdfsdfsdf-,Adolescenti,-testo%20text%20testo